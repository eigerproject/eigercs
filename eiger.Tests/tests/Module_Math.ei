include math

emitln(math.abs(-128))
emitln(math.abs(128))

emitln(math.sqrt(100))
emitln(math.sqrt(-100))

emitln(math.pow(2, -1))
emitln(math.pow(2, 10))

emitln(math.mod(2, 2))
emitln(math.mod(37, 3))

emitln(math.factorial(0))
emitln(math.factorial(5))

emitln(math.sin(0))