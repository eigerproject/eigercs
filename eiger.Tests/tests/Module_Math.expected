128
128
10
NaN
0.5
1024
0
1
1
120
0